test/testcafe/spec/Success_spec.js

```diff
@@ -0,0 +1,25 @@
+import IdentityPageObject from '../framework/page-objects/IdentityPageObject';
+import { ClientFunction, RequestMock } from 'testcafe';
+import success from '../../../playground/mocks/idp/idx/data/success'
+
+const mock = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx')
+  .respond(success)
+
+fixture(`Success Form`)
+  .requestHooks(mock)
+
+async function setup(t) {
+  const identityPage = new IdentityPageObject(t);
+  await identityPage.navigateToPage();
+  return identityPage;
+}
+const getPageUrl = ClientFunction(() => window.location.href);
+
+test(`should navigate to redirect link google.com after success`, async t => {
+  const identityPage = await setup(t);
+  await identityPage.fillIdentifierField('Test Identifier');
+  await identityPage.clickNextButton();
+  await t.expect(getPageUrl()).contains('https://httpbin.org/get');
```

Comment:
set redirect url to be something like `https://httpbin.org/get?stateToken=abc123` and verify the page url.

Reply:
testcafe had trouble verifying the url when we hit `https://httpbin.org/get?stateToken=abc123` (test just gets stuck without error) but was able to verify when it was `https://httpbin.org?stateToken=abc123`

Reply:
open a ticket to follow up?